src/TwigComponent/doc/index.rst

```diff
@@ -832,6 +832,140 @@ The ``with`` data is what's mounted on the component object.
 
     Embedded components *cannot* currently be used with LiveComponents.
 
+Passing blocks to embedded components
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.10
+
+    The `outerBlocks` variable has been added since 2.10.
+
+By combining nested components and embedded components very powerful structures can be created.
+There's one important thing to remember: **each embedded component, a.k.a. a component with
+blocks, inside a template, a.k.a. a nested component, starts its own template**.
+Any blocks that are available in your component template are *NOT* available *inside* the embedded component.
+
+Imagine this simple Alert component, which allows its content to be set by a higher component,
+for example a SuccessAlert component.
+
+.. code-block:: html+twig
+
+    {# templates/alert.html.twig #}
+    <div class="alert alert-{{ type }}">
+        {% block content %}{% endblock %}
+    </div>
+
+.. code-block:: twig
+
+    {# templates/successAlert.html.twig #}
+    {% component Alert with { type: 'success' } %}
+        I can override the content, but that's pretty dull.
+    {% endcomponent %}
+
+What if this SuccessAlert component wants to make the content of its nested Alert component configurable?
+Well, you can use the special `outerBlocks` variable for those cases. This makes it possible to refer
```

Comment:
In rst, unlike markdown, all technical words have 2 “ticks” around them, not just 1